tests/e2e_tests/cypress/e2e/basic-test/basic-test.cy.js

```diff
@@ -0,0 +1,12 @@
+describe('the most basic test', () => {
+  beforeEach(() => {
+    // TODO this probably should be written in some cypress configuration
```

Comment:
This TODO probably should be in tasks managment system ;)

Reply:
https://github.com/Problematy/goodmap/issues/6 
I hope that will suffice :)